.gitignore

```diff
@@ -89,3 +89,6 @@ shared/oval/*.ini
 
 # Ignore intermediary output XML files in the shared directory
 shared/output/*.xml
+
+# Ignore PCI_DSS_v3-1.pdf
+shared/transforms/pcidss/PCI_DSS_v3-1.pdf
```

Comment:
Imho `shared/transforms/pcidss/*.pdf` is more useful, but it is not very important